mindsdb_native/__init__.py

```diff
@@ -29,8 +29,12 @@
 try:
     from mindsdb_native.libs.data_sources.postgres_ds import PostgresDS
 except:
-    print("PostgresDS Datasource is not available by default. If you wish to use it, please install psycopg2 or mindsdb[extra_data_sources]")
+    print("PostgresDS Datasource is not available by default. If you wish to use it, please install 'pg8000' or mindsdb[extra_data_sources]")
 
+try:
+    from mindsdb_native.libs.data_sources.ms_sql_ds import MSSQLDS
+except:
```

Comment:
when trying to import it's better to do `except ImportError`, because `except:` will silently catch all exceptions that might occur in the imported module's `__init__.py`